easy-ui-react/src/EmptyStateCard/EmptyStateCard.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { ReactNode } from "react";
+import { Text, TextProps } from "../Text";
+import { Card } from "../Card";
+import { VerticalStack, VerticalStackProps } from "../VerticalStack";
+
+export type EmptyStateCardProps = Omit<
+  VerticalStackProps,
+  "children" | "as" | "reverseOrder"
+> & {
+  /**
+   * The children of the <EmptyStateCard> element. Should render
+   * `<EmptyStateCard.Header>`, `<EmptyStateCard.Body>`, and
+   * `<EmptyStateCard.Action>` at minimum.
+   */
+  children: ReactNode;
+};
+
+/**
+ * An `<EmptyStateCard />` is a styled container with a header, body, and action sections,
+ * designed to display relevant information when there is no nearby data to display.
+ *
+ * @remarks
+ * Can accept a subset of properties from the `<VerticalStack />` component to assist
+ * with custom content alignment.
+ *
+ * @example
+ * ```tsx
+ * <EmptyStateCard>
+ *  <EmptyStateCard.Header>
+ *    <EmptyStateCard.HeaderText>
+ *      Shipment Insurance
+ *    </EmptyStateCard.HeaderText>
+ *  </EmptyStateCard.Header>
+ *  <EmptyStateCard.Body>
+ *    <EmptyStateCard.BodyText>
+ *      Rest easy knowing if one of your customers orders is damaged, lost
+ *      in transit or stolen you are covered! Automatically add insurance to
+ *      all your shipments
+ *    </EmptyStateCard.BodyText>
+ *  </EmptyStateCard.Body>
+ *  <EmptyStateCard.Action>
+ *    <Button>Manage Insurance Settings</Button>
+ *  </EmptyStateCard.Action>
+ * </EmptyStateCard>
+ * ```
+ */
+export function EmptyStateCard(props: EmptyStateCardProps) {
+  const {
+    children,
+    gap = "2",
+    align = "space-between",
+    inlineAlign = "start",
+    ...restVerticalStackProps
+  } = props;
+
+  return (
+    <Card background="primary.800" borderRadius="lg" padding="5" boxShadow="1">
```

Comment:
checked with design about `boxShadow`; mockups have value that doesn't map to one of our card values.